tests/py_test.py

```diff
@@ -130,6 +130,8 @@ def asserter(stmt):
         invsum += int(v)
     asserter(invsum == 10)
 
+    asserter(monster.InventoryAsNumpy().sum() == 10)
```

Comment:
Yep - just force pushed change that should assert the correct error is raised if numpy does not exist, and I run the python tests twice (once w/o numpy, and once after numpy is installed).